Gemfile.lock

```diff
@@ -499,38 +502,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
-      coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+    path_expander (1.1.1)
+    pg (1.5.4)
```

Comment:
### [pg](https://github.com/ged/ruby-pg)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.3 → 1.5.4

[_[change-log](https://github.com/ged/ruby-pg/blob/master/History.md), [source-code](https://github.com/ged/ruby-pg), [gem-diff](https://my.diffend.io/gems/pg/1.1.3/1.5.4)_]





<details>
<summary>Commits</summary>

A change of **772** commits. See the full changes on [the compare page](https://github.com/ged/ruby-pg/compare/v1.1.3...v1.5.4).

These are the first 10 commits:
- (fc806b8) [Move specs to "OS thread support" into async file](https://github.com/ged/ruby-pg/commit/fc806b86d3c264daaf2350c890f419d138a81018)
- (a317078) [Small fix in comment at conn.flush](https://github.com/ged/ruby-pg/commit/a317078f5642b35f0c628f769ac0f7ea916bb0db)
- (4e65702) [Merge pull request #459 from larskanis/timeout](https://github.com/ged/ruby-pg/commit/4e657028b5ffe07f5b1df3adc981246a076621bb)
- (40f4609) [Bundle all conn.copy_data related specs together](https://github.com/ged/ruby-pg/commit/40f46095234fda5550979fb31afe87743ef934a8)
- (ad80e04) [Remove out-commented debug outputs](https://github.com/ged/ruby-pg/commit/ad80e04ec6dfee37628a7e0947b383b9fa8c714d)
- (db506d1) [Don't flush at each put_copy_data call, but flush at get_res…](https://github.com/ged/ruby-pg/commit/db506d11c50247c0058993742697403260d0ad64)
- (8141b8d) [Merge pull request #462 from larskanis/flush](https://github.com/ged/ruby-pg/commit/8141b8d32e1f519201d358d7b2d4f35839821877)
- (ae8361a) [Avoid old style K&R declaration syntax](https://github.com/ged/ruby-pg/commit/ae8361a00d0ef0cd261e3fa94e2bca41ed54a2a2)
- (91b00d0) [Set connection attribute of PG::Error](https://github.com/ged/ruby-pg/commit/91b00d02072d67698b01af767d8a76bb95c6e9b9)
- (26cd0f7) [Revert "April is coming!"](https://github.com/ged/ruby-pg/commit/26cd0f750ffb266a956828ad43e99eafecafcfbf)


</details>